### DISCUSSION TITLE
Detailed instructions for exporting model

### DISCUSSION THREAD

**Discussion by ADR3-N** [voicepaw/so-vits-svc-fork#529] [Q&A] [answered]
I'm a first timer for anything related to deep learning, models, python, and all. Could someone explain to me in detail the exact steps needed (including code and any other dependencies) how to export my model? I have people who want it and I have not even the slightest idea of how to give it to them.

I'm running this exact fork in Python on Win10. 
The only model I have is the one I'm training, which contains 2 speakers.

I've been researching this for about 2 weeks, tried chat GPT, and can't seem to find out how to get it done. Thanks for any answers. I'll happily wear my dunce cap until I or someone else manages to help me figure this out!

**Comment by 34j** [comment]
The code for ONNX export was not working and was removed a long time ago, but you can be reference and impove it.

**Reply by ADR3-N** [reply]
Considering it's week two of downloading python and I was completely ignorant of it before, that's a tall order for me. Is there another way people have been uploading their models? What folders am I looking to copy to a zip?

**Reply by ADR3-N** [reply, marked as answer]
I received an answer to my question elsewhere. Someone said people are just uploading the G model from logs with the config file. If this is wrong, please correct me.
